test/suite/test_backup14.py

```diff
@@ -26,64 +26,37 @@
 # ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
 # OTHER DEALINGS IN THE SOFTWARE.
 
-import wiredtiger, wttest
+import wiredtiger
 import os, shutil
-from helper import compare_files
-from suite_subprocess import suite_subprocess
+from wtbackup import backup_base
 from wtdataset import simple_key
 from wtscenario import make_scenarios
 import glob
 
 # test_backup14.py
 # Test cursor backup with a block-based incremental cursor.
-class test_backup14(wttest.WiredTigerTestCase, suite_subprocess):
+class test_backup14(backup_base):
     conn_config='cache_size=1G,log=(enabled,file_max=100K)'
     dir='backup.dir'                    # Backup directory name
     logmax="100K"
     uri="table:main"
     uri2="table:extra"
     uri_logged="table:logged_table"
     uri_not_logged="table:not_logged_table"
-    full_out = "./backup_block_full"
-    incr_out = "./backup_block_incr"
+
     bkp_home = "WT_BLOCK"
     home_full = "WT_BLOCK_LOG_FULL"
     home_incr = "WT_BLOCK_LOG_INCR"
     logpath = "logpath"
-    nops=1000
-    mult=0
     max_iteration=7
-    counter=0
     new_table=False
-    initial_backup=False
 
     pfx = 'test_backup'
     # Set the key and value big enough that we modify a few blocks.
     bigkey = 'Key' * 100
     bigval = 'Value' * 100
 
-    #
-    # Set up all the directories needed for the test. We have a full backup directory for each
-    # iteration and an incremental backup for each iteration. That way we can compare the full and
-    # incremental each time through.
-    #
-    def setup_directories(self):
-        for i in range(0, self.max_iteration):
-            remove_dir = self.home_incr + '.' + str(i)
-
-            create_dir = self.home_incr + '.' + str(i) + '/' + self.logpath
-            if os.path.exists(remove_dir):
-                os.remove(remove_dir)
-            os.makedirs(create_dir)
-
-            if i == 0:
-                continue
-            remove_dir = self.home_full + '.' + str(i)
-            create_dir = self.home_full + '.' + str(i) + '/' + self.logpath
-            if os.path.exists(remove_dir):
-                os.remove(remove_dir)
-            os.makedirs(create_dir)
-
+    nops = 1000
```

Comment:
I'm not sure why this was moved to here from the group of initializations above.

Reply:
think it was a bad merge on my part, ill move it back 👍